### PR TITLE
Updated dotnet45 and dotnet452 (remove win2003)

### PR DESCRIPTION
**Pull request by ImperatorS79**
As asked in #428.

### CODE REVIEW

File: Engines/Wine/Verbs/dotnet45/script.js [hunk 1]
```diff
@@ -42,8 +42,11 @@ Wine.prototype.dotnet45 = function() {
     this.overrideDLL()
         .set("native", ["mscoree"])
         .do();
-        
-    this.windowsVersion("win2003");
-
+
```
Comment (plata) [inline Engines/Wine/Verbs/dotnet45/script.js]: Can we replace this by a warning?
Something like:
```Javascript
if (this.windowsVersion != "win2003") {
    print(tr("Windows version must be \"win2003\" such that {0} applications can work."), ".NET 4.5");
}
```

(untested)

Comment (ImperatorS79) [reply Engines/Wine/Verbs/dotnet45/script.js]: As the windows version is set to win7 at the beginning of the script, there is no need of the "if".

Comment (plata) [reply Engines/Wine/Verbs/dotnet45/script.js]: Ok, sorry I didn't see that. In any case, this has the same problem as setting "win2003" in the verb: it's not clear from the outside that the Windows version is changed. Apparently, "win7" is needed for the installation. I would therefore suggest the following:
1. store the Windows version
2. set to win7
3. install .Net
4. restore the Windows version
5. if restores Windows version != win2003, print warning

Comment (ImperatorS79) [reply Engines/Wine/Verbs/dotnet45/script.js]: {0} isn't replaced by .NET 4.5 for an obscure reason

Comment (madoar) [reply Engines/Wine/Verbs/dotnet45/script.js]: Should `.NET 4.5` be passed to `tr` or to `print`?

Comment (plata) [reply Engines/Wine/Verbs/dotnet45/script.js]: Yes to `tr`. I hope that works in JS.

@ImperatorS79 can you try
```Javascript
if (this.windowsVersion != "win2003") {
    print(tr("Windows version must be \"win2003\" such that {0} applications can work.", ".NET 4.5"));
}
```

Comment (ImperatorS79) [reply Engines/Wine/Verbs/dotnet45/script.js]: Everything seems to work now

File: Engines/Wine/Verbs/dotnet45/script.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ Wine.prototype.dotnet45 = function() {
     if (this.architecture() == "amd64") {
         throw "{0} cannot be installed in a 64bit wine prefix!".format("dotnet45");
     }
+    
+    var OSVersion = this.windowsVersion;
```
Comment (plata) [inline Engines/Wine/Verbs/dotnet45/script.js]: This works without `()`? I was expecting `this.windowsVersion()`.

File: Engines/Wine/Verbs/dotnet45/script.js [hunk 3]
```diff
@@ -43,7 +45,11 @@ Wine.prototype.dotnet45 = function() {
         .set("native", ["mscoree"])
         .do();
         
-    this.windowsVersion("win2003");
-
+    this.windowsVersion(OSVersion);
+    
+    if(OSVersion != "win2003") {
+     print(tr("{0} applications can have issues when windows version is not set to \"win2003\"", ".NET 4.5"));
```
Comment (plata) [inline Engines/Wine/Verbs/dotnet45/script.js]: Please adjust indentation to rest of the file.

File: Engines/Wine/Verbs/dotnet45/script.js [hunk 4]
```diff
@@ -43,7 +45,11 @@ Wine.prototype.dotnet45 = function() {
...
+    }
+    
     return this;
-}; 
+};
```
Comment (plata) [inline Engines/Wine/Verbs/dotnet45/script.js]: Can you remove the trailing whitespace?

File: Engines/Wine/Verbs/dotnet452/script.js [hunk 5]
```diff
@@ -13,6 +13,8 @@ Wine.prototype.dotnet452 = function() {
     if (this.architecture() == "amd64") {
         throw "{0} cannot be installed in a 64bit wine prefix!".format("dotnet452");
     }
+    
+    var OSVersion = this.windowsVersion;
```
Comment (plata) [inline Engines/Wine/Verbs/dotnet452/script.js]: This works without `()`? I was expecting `this.windowsVersion()`.

File: Engines/Wine/Verbs/dotnet452/script.js [hunk 6]
```diff
@@ -41,8 +43,12 @@ Wine.prototype.dotnet452 = function() {
...
         .set("native", ["mscoree"])
         .do();
-        
-    this.windowsVersion("win2003");
-
+      
+    this.windowsVersion(OSVersion);
+    
+    if(OSVersion != "win2003") {
+     print(tr("{0} applications can have issues when windows version is not set to \"win2003\"", ".NET 4.5.2"));
```
Comment (plata) [inline Engines/Wine/Verbs/dotnet452/script.js]: Please adjust indentation to rest of the file.

**Pull request merged**